src/iomei.cpp

```diff
@@ -4730,6 +4749,49 @@ bool MEIInput::ReadLabelAbbr(Object *parent, pugi::xml_node labelAbbr)
     return this->ReadTextChildren(vrvLabelAbbr, labelAbbr, vrvLabelAbbr);
 }
 
+bool MEIInput::ReadLayerDef(Object *parent, pugi::xml_node layerDef)
+{
+    LayerDef *vrvLayerDef = new LayerDef();
+
+    vrvLayerDef->ReadLabelled(layerDef);
+    vrvLayerDef->ReadNInteger(layerDef);
+    vrvLayerDef->ReadTyped(layerDef);
+
+    parent->AddChild(vrvLayerDef);
+    this->ReadUnsupportedAttr(layerDef, vrvLayerDef);
+    return this->ReadLayerDefChildren(vrvLayerDef, layerDef);
+}
+
+bool MEIInput::ReadLayerDefChildren(Object *parent, pugi::xml_node parentNode)
+{
+    assert(dynamic_cast<LayerDef *>(parent));
+
+    bool success = true;
+    pugi::xml_node current;
+    for (current = parentNode.first_child(); current; current = current.next_sibling()) {
+        if (!success)
```

Comment:
Just a suggestion, consider using a range-based for loop
for (pugi::xml_node current : parentNode.children())